src/mmw/js/src/analyze/tests.js

```diff
@@ -129,19 +129,35 @@ function pointsourceTableFormatter(categories) {
     });
 }
 
+function catchmentWaterQualityTableFormatter(categories) {
+    var collection = new coreModels.CatchmentWaterQualityCensusCollection(categories);
+
+    return collection.map(function(category) {
+        var nord = category.get('nord').toString(),
+            areaha = category.get('areaha') === null ? 'No Data' : category.get('areaha').toLocaleString(),
+            tn_tot_kgy = category.get('tn_tot_kgy') === null ? 'No Data' : (category.get('tn_tot_kgy')/category.get('areaha')).toLocaleString(),
+            tp_tot_kgy = category.get('tp_tot_kgy') === null ? 'No Data' : (category.get('tp_tot_kgy')/category.get('areaha')).toLocaleString(),
+            tss_tot_kg = category.get('tss_tot_kg') === null ? 'No Data' : (category.get('tss_tot_kg')/category.get('areaha')).toLocaleString();
```

Comment:
If `areaha` is `null` or 0, should this skip the other checks? I checked in the Node repl and the browser console and it looks like dividing an actual number by 0 or `null` returns infinity.